src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/controllers/AddAmendClaimantRepresentativeController.java

```diff
@@ -55,17 +60,32 @@ public class AddAmendClaimantRepresentativeController {
         @ApiResponse(responseCode = "500", description = "Internal Server Error")
     })
     public ResponseEntity<CCDCallbackResponse> aboutToSubmit(
-            @RequestBody CCDRequest ccdRequest,
-            @RequestHeader("Authorization") String userToken) {
-
-        if (!verifyTokenService.verifyTokenSignature(userToken)) {
-            log.error(INVALID_TOKEN, userToken);
-            return ResponseEntity.status(FORBIDDEN.value()).build();
-        }
+            @RequestBody CCDRequest ccdRequest) {
 
         CaseData caseData = ccdRequest.getCaseDetails().getCaseData();
-        addAmendClaimantRepresentativeService.setRepresentativeId(caseData);
+        addAmendClaimantRepresentativeService.addAmendClaimantRepresentative(caseData);
 
         return getCallbackRespEntityNoErrors(caseData);
     }
+
+    @PostMapping("/amendClaimantRepSubmitted")
+    @Operation(summary = "processes notice of change update after amending claimant representatives")
+    @ApiResponses(value = {
+        @ApiResponse(responseCode = "400", description = "Bad Request"),
+        @ApiResponse(responseCode = "500", description = "Internal Server Error")
+    })
+    public void amendRespondentRepSubmitted(
+            @RequestBody CallbackRequest callbackRequest,
+            @RequestHeader(HttpHeaders.AUTHORIZATION) String userToken) {
+
+        CaseDetails caseDetails = callbackRequest.getCaseDetails();
+        log.info("AMEND CLAIMANT REPRESENTATIVE SUBMITTED ---> "
+                + LOG_MESSAGE + callbackRequest.getCaseDetails().getCaseId());
+        try {
+            String currentUserEmail = userIdamService.getUserDetails(userToken).getEmail();
+            nocClaimantRepresentativeService.updateClaimantRepresentation(caseDetails, currentUserEmail);
```

Comment:
Is this correct? In nocClaimantRepresentativeService.updateClaimantRepresentation, the second argument looks to be the userToken